aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -432,7 +421,7 @@ mod test {
         let mut aggregator_data = AggregatorData::default();
 
         let aggregator = aggregator_data
-            .get_aggregator(aggregator_id_for_test(500), 500, &fake_resolver, false)
+            .get_aggregator(aggregator_id_for_test(500), 500)
```

Comment:
Cannot comment on above, but this test makes no sense anymore if we remove the flag? Let's remove it as well? I think there are a few tests like that as well which are duplicates without the flag.

Reply:
Removed unnecessary test cases.